src/EditorFeatures/Core/Implementation/Suggestions/SuggestedActionSetComparer.cs

```diff
@@ -18,7 +18,7 @@ public SuggestedActionSetComparer(SnapshotPoint? targetPoint)
         private static int Distance(Span? textSpan, SnapshotPoint? targetPoint)
         {
             // If we don't have a text span or target point we cannot calculate the distance between them
-            if (textSpan == null || !textSpan.HasValue || targetPoint == null || !targetPoint.HasValue)
+            if (!textSpan.HasValue || !targetPoint.HasValue)
```

Comment:
Me too, and it's what I did first. But the prevailing style of the file is the opposite and that takes precedence outside of a style-changing PR.